Pocket/pocket-ios
# feat(Tabs) Bring the user back to their last-seen tab on restart

## Summary
When the app is backgrounded, write the currently selected tab to UserDefaults.
When the app is loading, read that value and set the respective tab to active.

## References 
https://getpocket.atlassian.net/browse/IN-1266

## Implementation Details
Almost all work done in MainViewModel. Added 3 helper functions for reading, writing and clearing the entry from UserDefaults. Keys added to the UserDefaults extension.

## Test Steps
Open the app. A fresh open should default to Home.

Navigate to any tab and background the app. Either wait for the OS to kill the app or quit the app yourself.
Relaunching the app should leave you on the tab you last saw.

## PR Checklist:
